### PR TITLE
adding job submit ENVIRONMENT and minor improvement to runWorkflow

### PR DESCRIPTION
**Pull request by antgonza**
adding job submit ENVIRONMENT and minor improvement to runWorkflow

### CODE REVIEW

File: qiita_db/processing_job.py [hunk 1]
```diff
@@ -880,12 +880,30 @@ def submit(self, parent_job_id=None, dependent_jobs_list=None):
...
+                self._set_status('running')
+                qdb.sql_connection.TRN.commit()
+
+            create_nested_path(job_dir)
+            cmd = (f'{plugin_env_script}; {plugin_start_script} '
+                   f'{url} {self.id} {job_dir}')
+            stdout, stderr, return_value = _system_call(cmd)
+            if return_value != 0 or stderr != '':
+                self._set_error(stderr)
+            job_id = stdout
```
Comment (ElDeveloper) [inline qiita_db/processing_job.py:900]: Does this string need to be stripped for new lines or something like that?

**Review by ElDeveloper — APPROVED** [review]
Just one question/comment.

File: qiita_db/processing_job.py (hunk 1 above)
Comment (antgonza) [reply qiita_db/processing_job.py:900]: Excellent question! IMOO no. Basically, the idea is to display whatever information the process creating the job returns.

Comment (ElDeveloper) [reply qiita_db/processing_job.py:900]: Got it, should be used to display to the end-user not to decode a return value. Makes sense.

**Pull request merged**